src/redux/settings/erc20Manager/tokens.js

```diff
@@ -93,6 +93,7 @@ export const formTokenLoadMetaData = async (token: TokenModel, dispatch) => {
 
   dispatch({type: TOKENS_FORM_FETCH, end: true})
 
+  // TODO @ipavlenko: check a list of existent tokens, not only ETH
```

Comment:
Ok, TODO removed
But we really have no duplicate check  for address on the validation phase, I can show you.